packages/react-query/src/internals/getQueryKey.ts

```diff
@@ -116,3 +117,8 @@ export function getQueryKey<
   const queryKey = getQueryKeyInternal(path, input, type ?? 'any');
   return queryKey;
 }
+
```

Comment:
add a TODO here - i think there's some cleanup that can be done here on next branch